examples/aidp_openai_demo/scripts/load_support_tickets.py

```
@@ -0,0 +1,376 @@
+  }}
+]"""
+
+    payload = {
+        "model": LLM_MODEL,
+        "messages": [{"role": "user", "content": prompt}],
+        "temperature": 0.8,
+        "max_tokens": 8000
+    }
+    
+    print(f"Generating {num_tickets} synthetic support tickets using {LLM_MODEL}...")
+    response = requests.post(NVIDIA_LLM_URL, headers=headers, json=payload)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)